MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -45,6 +45,20 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      */
     @objc public var abbreviationPriority: Int = NSNotFound
     
+    /**
+     An array indicating which directions you can go from a lane (left, right, or straight).
+     
+     If the value is `[.left", .straight]`, the driver can go straight or left from that lane. This is only set when the `component` is a lane.
+     */
+    @objc public var directions: LaneIndication
```

Comment:
This should be an array of `LaneIndication`.

Reply:
Let's also rename this `indications` to [follow other patterns](https://github.com/mapbox/MapboxDirections.swift/blob/42ec86382b180277394e3c1717c6c1992f86b84a/MapboxDirections/MBLane.swift#L11) in this lib.

Reply:
@bsudekum Yep, it's already an array... 😸